### PR TITLE
Increased rt_retry_delay for replication_upgrade test

### PR DESCRIPTION
**Pull request by korry8911**
@javajolt @macintux 
I added some code to set the rt_rety_delay to 5 seconds before the test is run and then reset rt_rety_delay to the previous value.

### CODE REVIEW

File: tests/replication_upgrade.erl [hunk 1]
```diff
@@ -4,6 +4,13 @@
 -include_lib("eunit/include/eunit.hrl").
 
 confirm() ->
+
+    OldDelay = rt_config:get(rt_retry_delay, 1000),
+    lager:info("OldDelay is ~p", [OldDelay]),
+    rt_config:set(rt_retry_delay, 5000),
```
Comment (hazen) [inline tests/replication_upgrade.erl]: I'm a bit concerned about hard-coding a timeout.  How about something a bit more flexible, but works just the same: `OldDelay * 5`

File: tests/replication_upgrade.erl [hunk 2]
```diff
@@ -4,6 +4,13 @@
 -include_lib("eunit/include/eunit.hrl").
 
 confirm() ->
+
+    OldDelay = rt_config:get(rt_retry_delay, 1000),
+    lager:info("OldDelay is ~p", [OldDelay]),
```
Comment (hazen) [inline tests/replication_upgrade.erl]: How about something a bit more explicit, like "Original rt_retry_delay is..."

File: tests/replication_upgrade.erl [hunk 3]
```diff
@@ -4,6 +4,13 @@
...
+    rt_config:set(rt_retry_delay, 5000),
+    NewDelay = rt_config:get(rt_retry_delay),
+    lager:info("NewDelay is ~p", [NewDelay]),
```
Comment (hazen) [inline tests/replication_upgrade.erl]: Matching comment for rt_retry_delay

File: tests/replication_upgrade.erl [hunk 4]
```diff
@@ -93,4 +100,6 @@ confirm() ->
                                rt:log_to_nodes(Nodes, "Replication with upgraded node: ~p", [Node]),
                                replication:replication(ANodes, BNodes, true)
                        end, NodeUpgrades),
+    lager:info("Resetting Delay to is ~p", [OldDelay]),
```
Comment (hazen) [inline tests/replication_upgrade.erl]: Again rt_retry_delay instead of "Delay"

**Review by hazen — CHANGES_REQUESTED** [review]
Just a few tweaks to comments and values

**Comment by hazen** [comment]
FYI - There are multiple delays so we just wanted to specify which one

**Comment by korry8911** [comment]
should look better now @javajolt

**Comment by hazen** [comment]
:+1:

**Review by hazen — APPROVED** [review]

**Pull request merged**